### ISSUE TITLE
JEI integration: Direct recipe pasting from the worker's JEI recipes

### ISSUE THREAD

**Issue by pantsguyapp** [ldtteam/minecolonies-features#630]
### Prerequisites 

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [X] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version:
- MineColonies Version:
- Related Mods and their Versions:

## Is your feature request related to a problem?

JEI allows you to search for recipes for one specific items, but you need to be in the normal recipe GUI and not the worker one in order to paste the recipe into the teaching recipe grid.

## Enhancement or Change description

Add in a "filling function" in worker's GUI that you can just paste it in the grid.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by SivartMcDorf** [comment]
having an issue with this lately as well

**Comment by SivartMcDorf** [comment]
Pretty sure this one is done isn't it.

**Comment by uecasm** [comment]
No, it's not.  You do still need to click through to the vanilla recipe before you can click the Plus.  There are technical reasons for why it works this way, although they're not insurmountable, just low priority since it's only one extra click away.
